api/v1alpha1/selfnoderemediationconfig_types.go

```diff
@@ -46,8 +46,7 @@ type SelfNodeRemediationConfigSpec struct {
 	// node will likely lead to data corruption and violation of run-once semantics.
 	// In an effort to prevent this, the operator ignores values lower than a minimum calculated from the
 	// ApiCheckInterval, ApiServerTimeout, MaxApiErrorThreshold, PeerDialTimeout, and PeerRequestTimeout fields.
-	// +kubebuilder:validation:Minimum=0
-	// +kubebuilder:default=180
+	// +kubebuilder:validation:Minimum=1
```

Comment:
[Context] I've added this change in order to differentiate between 0 value which is just the default (when field is empty) and 0 value field in by the user (This differentiation was needed in the webhook).

>in theory at least this can be considered as an API change. It makes CRs with 0 values invalid...

That's a good point, after some thinking I think we are still in the clear though.

Here is my line of thought:
- A 0 value of an older version would cause the removal of `SafeTimeToAssumeNodeRebootedSeconds` .
- So IIUC the risk here is a user missing  `SafeTimeToAssumeNodeRebootedSeconds` (that was set to 0)  after an upgrade 
- since I think both the risk is low and the consequences of this risk materializing are minor, I think we can go ahead with this change.